### PR TITLE
doc/rbd: tweaks for the LIO iSCSI gateway

### PR DESCRIPTION
**Pull request by dillaman**
Fixes: http://tracker.ceph.com/issues/21763
Signed-off-by: Jason Dillaman <dillaman@redhat.com>

### CODE REVIEW

**Comment by dillaman** [comment]
jenkins render docs

File: doc/rbd/iscsi-initiator-linux.rst [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-------------------------------------------------
+-------------------------
+iSCSI Initiator for Linux
+-------------------------
 
 **Prerequisite:**
 
--  Package ``iscsi-initiator-utils-6.2.0.873-35`` or newer must be
-   installed
+-  Package ``iscsi-initiator-utils`` or newer must be installed
```
Comment (trociny) [inline doc/rbd/iscsi-initiator-linux.rst]: @dillaman I am not sure what "newer" then means.

File: doc/rbd/iscsi-initiator-linux.rst [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 
--  Package ``device-mapper-multipath-0.4.9-99`` or newer must be
-   installed
+-  Package ``device-mapper-multipath`` or newer must be installed
```
Comment (trociny) [inline doc/rbd/iscsi-initiator-linux.rst]: The same about "newer".

**Comment by dillaman** [comment]
@trociny thanks -- update pushed

**Comment by trociny** [comment]
@ceph-jenkins retest this please

**Comment by trociny** [comment]
@ceph-jenkins retest this please

**Pull request merged**